python-scripts/src/main/python/dataproducts/services/misc/druid_job_submitter.py

```diff
@@ -51,19 +51,21 @@ def interpolate_config(self, report_config):
         print('String interpolation for the report config completed')
         return report_config_str
 
-    def check_schedule(self, reportSchedule,report_id):
-        if reportSchedule == 'DAILY' :
+    def check_schedule(self, report_schedule, report_id, interval_slider):
+        if report_schedule == 'DAILY':
             return True
-        elif reportSchedule == 'WEEKLY' :
-            if date.today().weekday() == 0:
+        elif report_schedule == 'WEEKLY':
+            interval_slider = int(interval_slider) if interval_slider is not None else 0
```

Comment:
interval_slider should be less than 7 for weekly. If the value is more than 7 ignore it